### PR TITLE
[Merged by Bors] - feat: add named paths to random type (CT-635)

### PR DESCRIPTION
**Pull request by amandasteinhauer**
**Fixes or implements CT-635**

### Brief description. What is this change?

Add typing for random editor v2 that uses path names instead of number of paths

### CODE REVIEW

File: packages/base-types/src/node/randomV2.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { NodeType } from './constants';
```
Comment (viniciusdacal) [inline packages/base-types/src/node/randomV2.ts:1]: I think you also need to import and export this on the index.ts file in the same folder

**Review by viniciusdacal — APPROVED** [review]

**Comment by amandasteinhauer** [comment]
bors cancel

**Pull request closed without merging**